### DISCUSSION TITLE
Tout comprendre sur les Signals | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#11] [General]
# Tout comprendre sur les Signals | Bonjour Angular

Tout comprendre sur les Signals dans Angular

https://apprendre.bonjour-angular.com/articles/tout-comprendre-sur-les-signals/

**Comment by PaulBarbieux** [comment]
Je suis en apprentissage d'Angular, et la partie "Le problème" m'a bien aidé à comprendre l'utilité de Signal. Merci.
